lib/src/widgets/auth_card.dart

```diff
@@ -360,9 +364,10 @@ class _LoginCard extends StatefulWidget {
       this.hideForgotPasswordButton = false,
       this.hideSignUpButton = false,
       this.loginAfterSignUp = true,
-      this.hideProvidersTitle = false})
+      this.hideProvidersTitle = false,
+      this.customRegisterItems})
       : super(key: key);
-
+  final List<ItemBuilder>? customRegisterItems;
```

Comment:
maybe the name `customLoginTextFields` is more descriptive together with the `LoginTextField` class name.